### ISSUE TITLE
Since v3.0.0-alpha.8 and also the newest v3.0.0, my form is submitted, when I flip between the months.

### ISSUE THREAD

**Issue by ThomasTr** [nathanreyes/v-calendar#1272]
Rolled back to v3.0.0-alpha.7, which is working fine for me.
I can't figure out what triggers the submission of the form.

This is my implementation:
````
<template>
    <div>
        <date-picker v-model="value" locale="de">
            <template v-slot="{ inputValue, inputEvents }">
                <input
                    class="form-control"
                    :id="id"
                    :name="name"
                    :value="inputValue"
                    v-on="inputEvents"
                />
            </template>
        </date-picker>
    </div>
</template>

<script>
    import {DateTime}   from 'luxon';
    import {DatePicker} from 'v-calendar';
    import 'v-calendar/dist/style.css';

    export default {
        name: "datepicker",
        components: {DatePicker},
        props: {
            id: {
                type: String,
                required: true
            },
            name: {
                type: String,
                required: true
            },
            date: {
                type: String,
                required: false
            },
        },
        data() {
            return {
                value: new Date,
            }
        },
        mounted() {
            if (undefined !== this.date) {
                this.value = DateTime.fromFormat(this.date, 'y-MM-dd').toJSDate();
            }
        },
        computed: {
            formDate() {
                return DateTime.fromJSDate(this.value).toFormat('dd.MM.y');
            }
        }
    }
</script>
````

**Comment by ioannisgiannis** [comment]
Same issue for me.
The problem is that months navigation buttons have no "type" attribute. 
Buttons should have type="button" attribute. 
If no "type" attribute exists, button causes form submition.

Fix please =)

**Comment by nathanreyes** [comment]
Fix published in 3.0.1

**Comment by ioannisgiannis** [comment]
Hello,

After upgrading from v3.0.0 to v3.0.1 the npm install failed.
I get the following error:
![image](https://user-images.githubusercontent.com/46316269/225864756-d0652ea5-9f9d-44b4-80d9-14fdd0414e48.png)

vue.js: v3.2.47
node.js: v16.19.0
